infrastructure/lambda/template.yaml

```diff
@@ -230,6 +230,7 @@ Resources:
                 - ssm:GetParameter
               Resource:
                 - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/OrdnanceSurveyAPIURL"
+                - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/AuthRequestKmsEncryptionKeyId"
```

Comment:
☝️ This SSM param is invoked in the shared `AddressSessionService` constructor which now has a `JWTDecrypter`. This param read policy is now required on all lambdas that use the service. This will be refactored out in a subsequent task